apps/web/app/_api/mutations/useItemDraw.ts

```diff
@@ -0,0 +1,16 @@
+import { useMutation } from '@tanstack/react-query';
+
+import { fetcher } from '../fetcher';
+import { ItemDrawResponse } from '../types/draw';
+
+const itemDraw = (): Promise<ItemDrawResponse> => {
+  return fetcher.post<ItemDrawResponse>('items/draw', { json: '' });
+};
```

Comment:
이쪽은 이렇게만 해도 타입 추론 잘 되지 않나염? json도 굳이 안 넣어도 될 것 같아여

```suggestion
const itemDraw = () => fetcher.post<ItemDrawResponse>('items/draw', { json: '' });
```

Reply:
마자여 요렇게 해도 돼여 수정해서 머지하겠습니당~